src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,47 @@
+package qna.domain;
+
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class Answers {
+
+    @OneToMany(mappedBy = "question")
+    private List<Answer> answers;
+
+    public Answers() {
+        answers = new ArrayList<>();
+    }
+
+    public boolean isIdenticalWriter(User loginUser) {
+
+        long count = answers.stream()
+                .filter(answer -> !answer.isOwner(loginUser))
+                .count();
+
+        return count == 0;
+    }
```

Comment:
`stream` api 를 활용하시면 같은 로직인데 다음과 같이 표현이 가능할 것 같아요 😆 
그리고 라인도 의미있는 부분에서 구분해주시면 더 좋을 것 같아요 👍
참고만해주세요!!🙇‍♂️🙏
```suggestion
    public boolean isIdenticalWriter(User loginUser) {
        return answers.stream()
                .allMatch(answer -> answer.isOwner(loginUser));
    }
```